pkg/asset/installconfig/powervs/client.go

```diff
@@ -1360,11 +1361,18 @@ func (c *Client) AddIPToLoadBalancerPool(ctx context.Context, lbID string, poolN
 	clbpmOptions = c.vpcAPI.NewCreateLoadBalancerPoolMemberOptions(lbID, *lbPool.ID, port, lbpmtp)
 	logrus.Debugf("AddIPToLoadBalancerPool: clbpmOptions = %+v", clbpmOptions)
 
-	lbpm, response, err = c.vpcAPI.CreateLoadBalancerPoolMemberWithContext(ctx, clbpmOptions)
-	if err != nil {
-		return fmt.Errorf("could not add the load balancer pool member and the response = %+v, err = %w", response, err)
-	}
-	logrus.Debugf("AddIPToLoadBalancerPool: CLBPMWC lbpm = %+v", lbpm)
+	return wait.PollUntilContextCancel(ctx,
+		time.Second*10,
+		false,
```

Comment:
Nit: Would you want this parameter to be `true`? Based on https://github.com/kubernetes/apimachinery/blob/v0.30.1/pkg/util/wait/poll.go#L24-L34, it would be ideal to try atleast once before context is cancelled and before 1 interval has passed.

Reply:
Hrm.  if the context is cancelled, then CAPI will fail regardless elsewhere.  Right @r4f4?  Yeah, I can change it, but having it try once won't save the installation.

Reply:
Agreed. I don't see any advantages of trying once the context is done.

Reply:
Looks like the bot merged this 45 minutes after my post.  Weird.